content/docs/user-guide/experiment-management/running-experiments.md

```diff
@@ -0,0 +1,300 @@
+# Running Experiments
+
+We explain how DVC codifies and executes experiments, setting their parameters,
+using multiple jobs to run them in parallel, and running them in queues, among
+other details.
+
+> 📖 If this is the first time you are introduced into data science
+> experimentation, you may want to check the basics in
+> [Get Started: Experiments](/doc/start/experiments/) first.
+
+## The pipeline
+
+DVC relies on <abbr>pipelines</abbr> that codify experiment workflows (code,
+<abbr>stages</abbr>, <abbr>parameters</abbr>, <abbr>outputs</abbr>, etc.) in a
+`dvc.yaml` file. These contain the commands to run the experiments.
+
+> 📖 See [Get Started: Data Pipelines](/doc/start/data-pipeline) for an intro to
+> this topic.  
+> Here we assume that there's already a working `dvc.yaml` file in the
+> <abbr>project</abbr>.
+
+[ug-pipeline-files]: /doc/user-guide/project-structure/pipelines-files
+
+### Running the pipeline
+
+You can run the pipeline using default settings with `dvc exp run`:
+
+```dvc
+$ dvc exp run
+```
+
+DVC keeps track of the dependency graph and runs only the stages with changed
+dependencies or missing outputs.
+
+> Example: for a pipeline composed of `prepare`, `train`, and `evaluate` stages,
+> if a dependency of `prepare` stage has changed, the downstream stages
+> (`train`, `evaluate`) are also run.
+
+### Running specific stages
```

Comment:
I'm still not clear why the guide should explain running specific stages, `--single-item`, `--all-pipelines`, and `--interactive` options below. There are too many different options about specifying pipelines or stages to explain them all (`--pipeline`, `--recursive`, etc.), and the explanations here don't add much beyond what would be in the command reference. If we want to explain pipeline concepts, we could have a pipelines guide. If we want to explain `exp run` and `repro` options separately, we could make changes in the command reference.  Why include these here?

Reply:
@dberenbaum the guide is where we want to go into detail about all the possibilities and how every option works, including some implementation details if needed.

> the explanations here don't add much beyond what would be in the command reference

We plan to remove those details from the cmd ref (only leave short descriptions and link here) but it's not included in this PR (should follow immediately).

Reply:
That said I'm also not 100% satisfied with this guide but I want to merge it and create a new issue with a list of things to fix from it (e.g. regarding mixing in the pipeline concept) so let's keep this comment unresolved for now.